src/DownloadFormat/DownloadFormat.styles.ts

```diff
@@ -0,0 +1,74 @@
+import styled, { css } from 'styled-components';
+
+interface OptionProps {
+  variant?: string;
+}
+
+export const FormatOption = styled.div<OptionProps>`
+  text-align: center;
+  font-weight: bold;
+  height: 36px;
+  font-size: 12px;
+  line-height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &:first-child {
+    border-radius: 5px 5px 0 0;
+  }
+
+  &:last-child {
+    border-radius: 0 0 5px 5px;
+  }
+
+  &:hover {
+    background-color: ${p => p.theme.colors.lightSkyBlue};
+    color: ${p => p.theme.colors.white};
+  }
+
+  &:active {
+    background-color: ${p => p.theme.colors.darkSkyBlue};
+  }
+
+  &:not(:last-child) {
+    border-bottom: 1px solid ${p => p.theme.colors.gray3};
+  }
+
+  ${p =>
+    p.variant === 'disabled' &&
```

Comment:
I don't think disabled should be a variant, I think the variant should be primary - with a separate disabled prop that is a boolean. We will need it for blocking propagation on methods at some point